src/main/java/org/schedulesdirect/api/Program.java

```diff
@@ -167,6 +168,22 @@ static private SimpleDateFormat getOrigAirDateFormat() {
 	}
 	
 	/**
+	 * Represents the entity type that is program represents
+	 * @author Ben Fisler
+	 *
+	 */
+	static public enum EntityType {
+		/**
+		 * An unknown value was provided; provide the value in a bug ticket for future inclusion
+		 */
+		UNKNOWN,
+		Show,
```

Comment:
I will add a static valueOf method then to translate the title case response from SD to uppercase enum values.